assembly/std/index.ts

```diff
@@ -41,11 +46,16 @@ export function call(
  *
  * Note: arguments serialization is to be handled by the caller and the callee.
  *
- * @param at -
- * @param functionName -
- * @param args -
+ * @param at - The address of the contract.
+ * @param functionName - The name of the function to call in that contract.
+ * @param args - The serialized arguments to send to the function to call.
```

Comment:
```suggestion
 * @param args - The arguments of the function we are calling.
```

(The "args" parameter doesn't need to be serialized. It is done in the " return env.localCall(at.toString(), functionName, args.serialize());"
)